packages/react-native-worklets/android/src/main/cpp/worklets/android/WorkletsModule.cpp

```diff
@@ -68,14 +44,33 @@ WorkletsModule::getForwardedRequestAnimationFrame() {
 
 void WorkletsModule::invalidateCpp() {
   javaPart_.reset();
+  messageQueueThread_.reset();
+  uiScheduler_.reset();
   workletsModuleProxy_.reset();
 }
 
 void WorkletsModule::registerNatives() {
   registerHybrid({
       makeNativeMethod("initHybrid", WorkletsModule::initHybrid),
+      makeNativeMethod(
+          "getBindingsInstaller", WorkletsModule::getBindingsInstaller),
       makeNativeMethod("invalidateCpp", WorkletsModule::invalidateCpp),
   });
 }
 
+jni::local_ref<BindingsInstallerHolder::javaobject>
+WorkletsModule::getBindingsInstaller() {
+  return jni::make_local(BindingsInstallerHolder::newObjectCxxArgs(
+      [&](jsi::Runtime &rnRuntime,
+          const std::shared_ptr<CallInvoker> &jsCallInvoker) {
+        workletsModuleProxy_ = std::make_shared<WorkletsModuleProxy>(
+            rnRuntime,
+            messageQueueThread_,
+            jsCallInvoker,
+            uiScheduler_,
```

Comment:
I understand that now we keep `messageQueueThread_` and `uiScheduler_` here because the initialization is "deferred". How about

```suggestion
            rnRuntime,
            std::move(messageQueueThread_),
            jsCallInvoker,
            std::move(uiScheduler_),
```

since we don't need these pointers here after `WorkletsModuleProxy` is created.

Reply:
I wanted not to keep them bound in `WorkletsModule.cpp` forever, and use `std::move` them to effectively reset them when constructing `WorkletsModuleProxy`.